modules/nf-core/ariba/run/tests/main.nf.test.snap

```diff
@@ -0,0 +1,18 @@
+{
+    "versions": {
+        "content": [
+            [
+                "versions.yml:md5,98363d30a08faf8ebda127cf44584ebe"
+            ]
+        ],
+        "timestamp": "2023-11-28T16:56:05.056901"
+    },
+    "illumina - fatsq": {
+        "content": [
+            [
+                
+            ]
```

Comment:
why is this empty ?

Reply:
Haha beat me to it @sateeshperi 

Here's the previous pytest outputs: https://github.com/nf-core/modules/blob/master/tests/modules/nf-core/ariba/run/test.yml

Reply:
I was having some trouble with the MD5s yesterday. But figured out what I was doing wrong. Working on matching the original test now!

Reply:
Okay! I believe I'm checking all expected outputs in the snapshot now: https://github.com/nf-core/modules/blob/e5459cf998c1c3ada532d48b0fb2365e0041e6b5/modules/nf-core/ariba/run/tests/main.nf.test.snap

I also added in the [contain tests](https://github.com/nf-core/modules/blob/e5459cf998c1c3ada532d48b0fb2365e0041e6b5/modules/nf-core/ariba/run/tests/main.nf.test#L48-L59) as well.

Reply:
Awesome @buehlere! this is learning experience for me as well. I'll defer to @sateeshperi 

@sateeshperi I'm wondering if `contains` can be given a list like in pytest. What would your recommendations be here? 

Haha I'm thinking of all the pytest tests that over use contains